data/fuel.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
kolkas to ner bet seip atsiras last updated data,tai yra jei kuro kaina nebuvo keista tos info nesaugai nes toks irasas egzistuoja,daba saugomi 3 irasai "Gegu\u017e\u0117s 26d. 13:00:00" bet kolkas sita pastebejima gali netvarkyt tik turek omeny

Reply:
ok